frontend/packages/zoo-api/src/client.ts

```diff
@@ -11,9 +12,30 @@ const cache = cacheExchange({
   },
 });
 
+
+function getToken(): string {
+    const value = localeStorage.getItem("the-zoo.api.token");
```

Comment:
Could you please at least:

1. Move [the constant declaration](https://github.com/kiwicom/the-zoo/blob/hangar/frontend/plugins/services/src/state/AppState.tsx#L9) in this file
2. Rename the constant name
3. Import this constant and rename it [where it’s used](https://github.com/kiwicom/the-zoo/blob/hangar/frontend/plugins/services/src/state/useSettings.ts)

Reply:
this is the exact function as it was before i removed severals commits ago :)

Reply:
done